src/doc/rustdoc/src/unstable-features.md

```diff
@@ -100,28 +100,16 @@ Note: Because of how `macro_rules` macros are scoped in Rust, the intra-doc link
 These features operate by extending the `#[doc]` attribute, and thus can be caught by the compiler
 and enabled with a `#![feature(...)]` attribute in your crate.
 
-### Documenting platform-/feature-specific information
+### `#[doc(cfg)]`: Recording what platforms or features are required for code to be present
```

Comment:
I'm not super happy with this wording, but I'm not sure how better to say it.

Reply:
Conditional documentation?

Reply:
No, the _documentation_ is usually unconditional (`cfg(any(doc, unix))`), but the _code_ is only present when the `cfg` is set.